src/main/kotlin/org/rust/lang/core/resolve2/ModCollector.kt

```diff
@@ -446,17 +448,18 @@ private sealed class ChildMod(val name: String, val hasMacroUse: Boolean) {
  * Have to pass [pathAttribute], because [RsFile.pathAttribute] triggers resolve.
  * See also: [RsMod.getOwnedDirectory]
  */
-private fun ChildMod.getOwnedDirectory(parentMod: ModData, pathAttribute: String?): VirtualFile? {
+private fun ChildMod.getOwnedDirectory(parentMod: ModData, pathAttribute: String?, includeMacroParent: VirtualFile?): VirtualFile? {
     if (this is ChildMod.File && name == RsConstants.MOD_RS_FILE) return file.virtualFile.parent
 
     val (parentDirectory, path) = if (pathAttribute != null) {
         when {
             this is ChildMod.File -> return file.virtualFile.parent
+            includeMacroParent != null -> includeMacroParent to pathAttribute
```

Comment:
It is a bit strange that now `ChildMod.getOwnedDirectory` and `RsMod.getOwnedDirectory` have different structure, though previously they mirror each other (have same set of conditions)